shared/store/enhancer.platform.desktop.js

```diff
@@ -1,11 +1,16 @@
 // @flow
 import DevTools from '../desktop/renderer/redux-dev-tools'
+import {batchedSubscribe} from 'redux-batched-subscribe'
 import {compose, applyMiddleware} from 'redux'
 import {enableStoreLogging} from '../local-debug'
+import {throttle} from 'lodash'
+
+const throttleNotify = throttle(notify => notify(), 200)
```

Comment:
we now update store subscribers in a throttled way. i explored using the react batched updates but this gave better runtime performance